charts/schema-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "schema-registry.configmap" . }}
```

Comment:
Question: how is this configuration used?

Reply:
These parameters are added as environment variables in the schema registry pod.